src/platform/testing/unit/mocha-setup.js

```diff
@@ -62,23 +65,20 @@ export default function setupJSDom() {
 
   global.Blob = window.Blob;
 
-  // from mocha-jsdom https://github.com/rstacruz/mocha-jsdom/blob/master/index.js#L80
-  function propagateToGlobal(window) {
-    /* eslint-disable */
-    for (const key in window) {
-      if (!window.hasOwnProperty(key)) continue;
-      if (key in global) continue;
-
-      global[key] = window[key];
-    }
-    /* eslint-enable */
-
-    // Mock fetch
-    // This was causing some tests to fail, so we'll have to loop back around to it later
-    // global.fetch = sinon.stub();
+  function copyProps(src, target) {
```

Comment:
Not sure exactly what this does but the `us-forms-system` had a jsdom set up similar to ours. The unit tests in the forms system wouldn't pass unless _it's_ set up function was run. I removed `propagateToGlobal` and it seems to work.

Reply:
Yeah, I'm not sure what the difference is, exactly. But if it works and the rest of the unit tests pass, then I guess it's good.